picard/ui/itemviews.py

```diff
@@ -267,6 +267,7 @@ def contextMenuEvent(self, event):
             if can_view_info:
                 menu.addAction(self.window.view_info_action)
             menu.addAction(self.window.browser_lookup_action)
+            menu.addAction(self.window.albums_search_action)
```

Comment:
albums here is plural but the class is named `AlbumSearchDialog`. I think it should be singular here too.